### PR TITLE
Review List Component #1

### PR DESCRIPTION
**Pull request by chrsbell**
Need to add dynamic text resizing to container size

### CODE REVIEW

File: client/src/components/ReviewList.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import StyledReview from './Review.jsx';
+import styled from 'styled-components';
+
+const FlexRow = styled.div`
+display: flex;
+flex-direction: row;
+`;
+
+const ReviewList = ( {reviews} ) => {
```
Comment (chrisalopez1337) [inline client/src/components/ReviewList.jsx:10]: Change to dynamically render reviews, not hard coded

**Review by chrisalopez1337 — CHANGES_REQUESTED** [review]
See comment, make sure your components can render dynamic sets of data, and not only hardcoded.

File: client/src/components/ReviewList.jsx (hunk 1 above)
Comment (chrsbell) [reply client/src/components/ReviewList.jsx:10]: https://github.com/keybox130/Reviews-Service/pull/4/commits/591f35faf10efd8be164ad4091f3b20ff885cb21

**Review by chrisalopez1337 — APPROVED** [review]
Changes look good!

**Pull request merged**